its/plugins/global-extension-plugin/pom.xml

```diff
@@ -29,6 +29,11 @@
       <version>${sonar.apiVersion}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.36</version>
```

Comment:
Better to define the version in `properties`?

Reply:
Only when you are reusing the same version multiple times IMO